### PR TITLE
Add encryption convention sample

### PR DESCRIPTION
**Pull request by dbelcham**
Adding a new sample to show unobtrusive encryption to replace the code removed from the sample in #3002 
Added a page to the Learning transport docs to show how to find and view messages.

### CODE REVIEW

File: samples/encryption/basic-encryption/sample.md [hunk 1]
```diff
@@ -54,10 +54,11 @@ snippet: enableEncryption
 
 This code indicates that encryption should be enabled.
 
-The key is then configured using
+The key is then configured using the `encryptionService` variable. This allows for property encryption using the special `WireEncryptedString` property seen in the `MessageWithSecretData.cs` file.
 
-snippet: ConfigureEncryption
+The following code snippet also shows the optional unobtrusive way to tell NServiceBus which properties to encrypt using the `encryptedPropertyConvention` parameter. In this example, all properties having a name that starts with `Encrypted` will have their value encrypted.
```
Comment (SimonCropp) [inline samples/encryption/basic-encryption/sample.md]: doesnt this PR add a new sample? if so why add "also shows..." to the existing sample?

Comment (dbelcham) [inline samples/encryption/basic-encryption/sample.md]: That shouldn't have made it in. Good catch.

File: samples/encryption/encryption-conventions/Core_5/Shared/EncryptionExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+#region ConfigureEncryption
```
Comment (SimonCropp) [inline samples/encryption/encryption-conventions/Core_5/Shared/EncryptionExtensions.cs]: i think this region can be inside the method. the fact that u r using an extenion method is not really relevant?

**Comment by dbelcham** [comment]
@SimonCropp it looks like you changed a lot of stuff in that last commit. Can you let me know what was the trigger for doing that so I can get things closer to the desired output next time?

**Comment by SimonCropp** [comment]
@dbelcham once I reviewed the content in relation to the other encryption sample I realized there was significant overlap. so I tweaked the content to make is easier to maintain and prevent unnecessary duplicate effort moving forward

**Pull request merged**